### PR TITLE
[Happy Path] Fix checking application readiness in HappyPath test

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR moves the first checking application readiness to ```try-catch``` block.

### CODE REVIEW

**Comment by dmytro-ndp** [comment]
@SkorikSergey: is it checked on OKD as well?

**Comment by SkorikSergey** [comment]
> @SkorikSergey: is it checked on OKD as well?

It checked on OKD on ```Che7-pr-happy-path-test-OKD job``` - https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Che7-pr-happy-path-test-OKD/908/.

**Comment by dmytro-ndp** [comment]
@SkorikSergey: thanks you for the answer.
IMHO, there is redundant messages `Application is not yet ready for use`, `Application is ready for use. App url:http://routerx5w1jh0-che.10.0.150.69.nip.io` in the log:
```
12:43:29   Validation of workspace start
12:43:30     ✓ Open workspace (1410ms)
12:44:41     ✓ Wait workspace running state (70106ms)
12:44:53     ✓ Wait until project is imported (12109ms)
12:44:53 
12:44:53   Language server validation
12:46:20     ✓ Java LS initialization (87526ms)
12:46:26     ✓ Error highlighting (5759ms)
12:46:31     ✓ Autocomplete (4837ms)
12:46:36     ✓ Suggestion (5163ms)
12:46:42     ✓ Codenavigation (6151ms)
12:46:42     - Yaml LS initialization
12:46:42 
12:46:42   Validation of workspace build and run
12:47:21     ✓ Build application (38436ms)
12:47:34 Application is not yet ready for use
12:47:35 Application is not yet ready for use
12:47:36 Application is not yet ready for use
12:47:37 Application is not yet ready for use
12:47:38 Application is not yet ready for use
12:47:39 Application is not yet ready for use
12:47:41 Application is ready for use. App url:http://routerx5w1jh0-che.10.0.150.69.nip.io
12:47:41     ✓ Run application (20696ms)
12:47:42     ✓ Check the running application (565ms)
12:47:42     ✓ Close preview widget (527ms)
12:47:49     ✓ Close the terminal running tasks (6640ms)
12:47:49 
12:47:49   Display source code changes in the running application
12:48:01     ✓ Change source code (12406ms)
12:48:38     ✓ Build application with changes (37050ms)
12:48:52 Application is not yet ready for use
12:48:53 Application is not yet ready for use
12:48:54 Application is not yet ready for use
12:48:55 Application is not yet ready for use
12:48:56 Application is not yet ready for use
12:48:57 Application is not yet ready for use
12:48:58 Application is not yet ready for use
12:48:59 Application is not yet ready for use
12:49:01 Application is ready for use. App url:http://routerx5w1jh0-che.10.0.150.69.nip.io
12:49:01     ✓ Run application with changes (22804ms)
12:49:02     ✓ Check changes are displayed (448ms)
12:49:04     ✓ Close preview widget (1880ms)
12:49:10     ✓ Close running terminal processes and tabs (6636ms)
```
It makes list of steps less clear. Do you really need to have it the logs by default?

**Comment by Ohrimenko1988** [comment]
On previous several planings, we discussed about making the output more informative. I have created issue for this task https://github.com/eclipse/che/issues/13686 .
So logging is good. But on my opinion, we should discuss and choose how it will look like and after that create some class which will implement logging of different layers. And only after that apply it for some particular cases.

**Comment by dmytro-ndp** [comment]
@SkorikSergey: what about usage `console.debug()` in _catch_ block?

**Comment by SkorikSergey** [comment]
> @SkorikSergey: what about usage `console.debug()` in _catch_ block?
> Will message be printed by default?

I will check it.

**Review by dmytro-ndp — APPROVED** [review]
Test output looks good now, IMHO https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/blue/organizations/jenkins/Single-Che-PR-check-E2E-Happy-path-tests-against-k8s-on-codenvy-slave9/detail/Single-Che-PR-check-E2E-Happy-path-tests-against-k8s-on-codenvy-slave9/263/:
Screenshot from 2019-09-02 15-13-01

File: e2e/pageobjects/ide/Ide.ts [hunk 1]
```diff
@@ -245,19 +245,17 @@ export class Ide {
 
     async waitApllicationIsReady(url: string,
         timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
-        let res = await axios.get(url);
+
         await this.driverHelper.getDriver().wait(async () => {
             try {
-                res = await axios.get(url);
+                let res = await axios.get(url);
```
Comment (Ohrimenko1988) [inline e2e/pageobjects/ide/Ide.ts]: I guess it should be "const" here instead of "let"

**Review by Ohrimenko1988 — APPROVED** [review]

**Pull request merged**